src/mixer.c

```diff
@@ -32,11 +36,11 @@ static mixer_t quadcopter_x_mixing =
 
 static mixer_t fixedwing_mixing =
 {
-  {M, S, S, S, 0, 0, 0, 0},
+  {S, S, M, S, 0, 0, 0, 0},
```

Comment:
Just curious what motivated this ordering? My initial thought would be to do aileron, elevator, rudder, throttle. Are these the RC channels or something? If it's a decision between matching RC inputs or making the offboard command setpoints intuitive, I would probably go with the latter, even if it means adding some remapping logic in the code here. Thoughts?